File: Wei.Web/app/model/account/UserAttribute.js
Ext.define('Wei.model.account.UserAttribute', {
    extend: 'Wei.model.Base',

    fields: [
        { type: 'int', name: 'id', persist: true },
        { type: 'string', name: 'name', persist: true },
        { type: 'string', name: 'displayname', persist: true },
        { type: 'string', name: 'tags', persist: true },
        { type: 'int', name: 'displayorder', persist: true },

    ]
});

File: Wei.Core/Domain/Users/UserAttribute.cs
namespace Wei.Core.Domain.Users
{
    public class UserAttribute: BaseEntity
    {
        /// <summary>
        /// 名称
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// 显示名称
        /// </summary>
        public string DisplayName { get; set; }

        /// <summary>
        /// 标签
        /// </summary>
        public string Tags { get; set; }

        /// <summary>
        /// 显示序号
        /// </summary>
        public int DisplayOrder { get; set; }
    }
}
